examples/mysql/helper.conf

```diff
@@ -1,5 +1,5 @@
 # SPIRE agent unix socket path
-agent_address = "/tmp/agent.sock"
+agent_address = "/tmp/spire-agent/public/api.sock"
```

Comment:
Do we need this config since its default value?

Reply:
This PR doesn't add the change to use a default value for that config, if that config property is not present, the startup process will throw an error.